Server/Processors/ServerMessageProcessor.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Domain.Game;
+using Domain.Messages;
+using System;
+using System.Collections.Generic;
+
+namespace Server.Processors
+{
+    public class ServerMessageProcessor : IMessageProcessor
+    {
+        public List<Message> ProcessConnectedMessage(Message message)
```

Comment:
Method expects ConnectionMessage type so it makes sense to change method signature to List<Message> ProcessConnectedMessage(ConnectionMessage message) (in IMessageProcessor too)

Reply:
done